data/data/bootstrap/baremetal/files/usr/local/bin/startironic.sh.template

```diff
@@ -152,11 +150,31 @@ $IPTABLES -t raw -A DHCP_IRONIC -j DROP
 {{end}}
 
 # Wait for images to be downloaded/ready
-podman wait -i 1000ms ipa-downloader
+#podman wait -i 1000ms ipa-downloader
 podman wait -i 1000ms coreos-downloader
-while ! curl --fail http://localhost/images/rhcos-ootpa-latest.qcow2.md5sum ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.initramfs ; do sleep 1; done
-while ! curl --fail --head http://localhost/images/ironic-python-agent.kernel ; do sleep 1; done
+
+# Write master network config to /tmp/nmstate for image_customization_controller to serve our merged images
+mkdir -p /tmp/nmstate
+{{range .PlatformData.BareMetal.Hosts}}
+    cat >/tmp/nmstate/{{.Name}}.yaml <<EOF
+{{.NetworkConfig}}
```

Comment:
I think we'll need to create secrets from the NetworkConfig, similar to how BMC credentials are handled:

https://github.com/openshift/installer/blob/master/pkg/asset/machines/baremetal/hosts.go#L29

That also means we'll have to consumer the manifest (not the install-config) data, since in theory a user can modify the manifests after running `create manifests` prior to running `create cluster`

So I'm wondering if we can just consume the secrets here (the manifests will get copied onto the bootstrap VM via ignition, like the BMH and BMC-secret manifests)

Reply:
Is that not what is happening here? https://github.com/openshift/installer/pull/5247/files

Reply:
Ah thanks I missed that, but I think the point remains that in startironic.sh we should consume those manifests, not the install-config data directly (I guess that could be addressed as a follow-up, but if we want to fix that now, this PR needs to be rebased onto https://github.com/openshift/installer/pull/5247 and the /tmp/nmstate stuff needs to consume the generated secrets)

Reply:
Gotcha.  I'll do that. I see it merged now.